lib/radbeacon/le_scanner.rb

```diff
@@ -1,10 +1,11 @@
 module Radbeacon
 
   class LeScanner
-    attr_accessor :duration
+    attr_accessor :duration, :options
 
     def initialize(duration = 5)
       @duration = duration
+      @options = {}
```

Comment:
Thanks for pointing that out, I didn't understand that `duration` and `hcitool_duration` did different things. Renaming it to make it obvious would be good.

When I saw the options hash I assumed it was along the lines of the [ruby idiom of passing an options hash](http://chriszetter.com/blog/2012/11/02/keyword-arguments-in-ruby-2-dot-0/). Seems like you want options to pass to the hcitool cli.

Depending on how you wan to manage that option, I might still make them first class attributes. I normally prefer attributes over a hash since they are discoverable on the instance of the object (like in irb). But ¯\_(ツ)_/¯